CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## unreleased
 
+## v4.4.0 - 2022.11.02
+
+* Update CSI driver for Kubernetes 1.25
+  [[GH-461]](https://github.com/digitalocean/csi-digitalocean/pull/461)
```

Comment:
Needs to say 464, not 461.

Reply:
464 is this PR, 461 is the one that updates the k8s deps 1.25

Reply:
Oh my bad, sorry.